docs/community/governance.md

```diff
@@ -90,36 +90,42 @@ One of their primary responsibilities is merging pull requests after all require
+### Working Group Responsibilities
 
-If you are an Area Expert, you are expected to be actively involved in any development, design, or contributions in your area of expertise.
+If you are a member of a Working Group, you are expected to be actively involved in any development, design, or contributions in the focus area of the WG.
+More information on the responsibilities of Working Groups can be found [here][wg],
+while current WG definitions and membership can be found [here][wg-definitions].
 
-If you are an Area Expert:
+If you are a Working Group member:
 
-1. **DO** assign the [correct labels][issue-process]
-1. **DO** assign yourself to issues labeled with your area of expertise
-1. **DO** code reviews for issues where you're assigned or in your areas of expertise.
-1. **DO** reply to new issues and pull requests that are related to your area of expertise
-  (while reviewing PRs, leave your comment even if everything looks good - a simple "Looks good to me" or "LGTM" will suffice, so that we know someone has already taken a look at it).
-1. **DO** make sure contributors are following the [contributor guidelines](../../.github/CONTRIBUTING.md).
-1. **DO** ask people to resend a pull request, if it [doesn't target `master`](../../.github/CONTRIBUTING.md#lifecycle-of-a-pull-request).
+1. **DO** triage and contribute to discussions in issues and PRs that have `WG-*` labels assigned
+1. **DO** regularly communicate with other members of your WG to coordinate decisions about
+   whether issues should move forward
+1. **DO** make decisions on whether or not issues should proceed forward with implementations or RFCs
+1. **DO** assign the [correct labels][issue-process] to issues
+1. **DO** assign yourself to issues labeled with your area of expertise when you plan to implement
+   the proposal in a reasonable timeframe
+1. **DO** code reviews for PRs where you're assigned or in your WG
+   (while reviewing PRs, leave your comment even if everything looks good - a simple "Looks good to me" or "LGTM" will suffice, so that we know someone has already taken a look at it).
+1. **DO** make sure contributors are following the [contributor guidelines](../../.github/CONTRIBUTING.md)
 1. **DO** ensure that contributors [write Pester tests][pester] for all new/changed functionality
 1. **DO** ensure that contributors [write documentation][docs-contributing] for all new-/changed functionality
-1. **DO** encourage contributors to refer to issues in their pull request description (e.g. `Resolves issue #123`).
-1. **DO** encourage contributors to create meaningful titles for all PRs. Edit title if necessary.
-1. **DO** verify that all contributors are following the [Coding Guidelines](../dev-process/coding-guidelines.md).
+1. **DO** ensure that contributors refer to issues in their pull request description (e.g. `Resolves issue #123`)
+1. **DO** ensure that contributors create meaningful titles for all PRs, editing their title if necessary
```

Comment:
Perhaps a bit more explanation that PR titles end up in the changelog, so titles should be user facing.